packages/docs/src/shared-components/appshell.ts

```diff
@@ -0,0 +1,350 @@
+import { html } from 'lit';
+import { getTranslation } from '../translations.js';
+
+/**
+ * Create a footer for all stories
+ */
+export const createFooter = () => html`
+  <!-- footer -->
+  <style>
+    .synergy-footer-demo {
+      container-type: inline-size;
+    }
+
+    .footer-content {
+      align-items: center;
+      display: flex;
+      flex-direction: row;
+      gap: var(--syn-spacing-x-large);
+      justify-content: space-between;
+      padding: var(--syn-spacing-large) var(--syn-spacing-2x-large);
+    }
+
+    .copyright {
+      color: var(--syn-typography-color-text);
+      font: var(--syn-body-small-bold);
+      margin: 0;
+    }
+
+    .link-wrapper {
+      display: flex;
+      gap: var(--syn-spacing-large);
+      justify-content: center;
+      list-style: none;
+      margin: 0;
+      padding: 0;
+    }
+
+    @container (max-width: 768px) {
+      .link-wrapper {
+        flex-direction: column;
+        gap: var(--syn-spacing-small);
+      }
+
+      .footer-content {
+        align-items: flex-start;
+        flex-direction: column;
+        padding: var(--syn-spacing-large) var(--syn-spacing-medium);
+      }
+    }
+  </style>
+  <footer class="synergy-footer-demo">
+    <div class="footer-content">
```

Comment:
From what I see from the applied styles, we may get away with the following markup:

```html
<footer class="synergy-footer-demo">
  <nav class="syn-footer-navigation-items">
    <a class="syn-link syn-link--medium syn-link-quiet" href="HREF">LINK/a>
    <a class="syn-link syn-link--medium syn-link-quiet" href="HREF">LINK/a>
    <a class="syn-link syn-link--medium syn-link-quiet" href="HREF">LINK/a>
    <a class="syn-link syn-link--medium syn-link-quiet" href="HREF">LINK/a>
  </nav>
  <p class="copyright">&copy; 2024 SICK AG</p>
</footer>
```

This would remove some unneeded markup from this function and also ease the css, as intermediate styling for the now unneeded li could be dropped in favor of something like:

```css
.syn-footer-navigation-items {
  ... Stuff from link-wrapper here, container stuff, too
}
```

What do you think?

Reply:
I didn't had the `<div class="footer-content"/>` in my previous version without the container query. But with container query we unfortunately need it..  😕 
The element, where the "container-type" is added, cannot be targeted for container queries
If we have the `<footer />` as container, we can't use it for changes like following:

```
@container (max-width: 768px) {
  footer {
      // style different if smaller
  }
}
```

Whether we leave out the `<ul />` and `<li />` we can discuss about it. You are right, that it is much cleaner without it. 
I did it, because I had a look at the `footer` components from others and they often wrap the `<a />`s into `<ul />`  and `<li/>`
( e.g. https://designsystem.digital.gov/components/footer/, https://edisondesignsystem.com/footer?tab=2)

What do you mean? Could it have any accessibility advantages?

Reply:
We've had the idea to add a link list component (Figma) and CSS styles for this <ul> <li> thing.
Maybe we refactor the footer link list later with it?